src/map/script.hpp

```diff
@@ -1898,6 +1898,23 @@ enum e_hat_effects {
 	HAT_EF_MAX
 };
 
+/**
+ * Player blocking actions related flags.
+ */
+enum pcblock_action_flag {
```

Comment:
Prefix the enum to match naming standard.

```suggestion
enum e_pcblock_action_flag {
```